dev/source/docs/setting-up-sitl-on-linux.rst

```diff
@@ -36,9 +36,6 @@ There is also a linked video below showing how to do the setup.
 .. youtube:: pJGFkZmGV6o
     :width: 100%
 
-.. include:: building-setup-linux.rst
```

Comment:
Hum, I don't know about this changes, I add this because people were complaining about unclear instruction to use SITL. I guess we need a deeper refactor on setting up the environnement

Reply:
The difference is that this includes the instructions for building on SITL within this page, while @stephendade just links to them. 

I have a slight preference towards linking to them (because there is likely to be some material in those docs that isn't as relevant in this doc) but either way is OK.

Reply:
I think linking makes it clearer to the user that we're referencing a different section of the documentation and keeps it consistent to the Windows SITL page (which also links) . Additionally makes this page shorter.

Reply:
:-)